index.html

```diff
@@ -101,7 +101,76 @@ <h3>Lecture</h3>
       </div>
     </div>
   </section>
-
+  <section class="artists">
+    <h2>Featured Artists</h2>
```

Comment:
- [ ] (OPTIONAL) Good job creating this section dynamically but your mobile design does not match the required design. Also in the mobile view, only 2 artists should be displayed together with a `more` button and when a user clicks on the `more` button, the remaining speakers are then displayed. It would be great if you could implement this functionality.
- Your design:
![Annotation 2022-03-11 063350](https://user-images.githubusercontent.com/85113567/157810307-3211fb2a-de86-4393-984f-8daa8b6a4110.jpg)
- Required design:
![Annotation 2022-03-11 063332](https://user-images.githubusercontent.com/85113567/157810325-b7632a22-b9a5-4402-bebf-599419271b26.jpg)

Reply:
Noted this comment and I will implement it later